src/views/catalog/CreateFromInstanceTypes/components/AddBootableVolumeButton/AddBootableVolumeButton.tsx

```diff
@@ -17,6 +17,9 @@ export type AddBootableVolumeButtonProps = {
 const AddBootableVolumeButton: FC<AddBootableVolumeButtonProps> = ({ buttonVariant }) => {
   const { t } = useKubevirtTranslation();
   const { createModal } = useModal();
+
+  const sourceNamespace = isUpstream ? KUBEVIRT_OS_IMAGES_NS : OPENSHIFT_OS_IMAGES_NS;
+
   const [canCreateVolume] = useAccessReview({
     group: DataSourceModel.apiGroup,
     namespace: isUpstream ? KUBEVIRT_OS_IMAGES_NS : OPENSHIFT_OS_IMAGES_NS,
```

Comment:
line 21 is the same?

Reply:
Ah yeah. I solved a conflict there. Fixed